src/auth.ts

```diff
@@ -90,36 +85,36 @@ export interface NamedIdentityType {
 };
 
 interface AuthRequestType {
-   jti: string;
-   iat: number;
-   exp: number;
-   iss: null | string;
-   public_keys: string[];
-   domain_name: string;
-   manifest_uri: string;
-   redirect_uri: string;
-   version: string;
-   do_not_include_profile: boolean;
-   supports_hub_url: boolean;
-   scopes: string[]
+  jti: string;
+  iat: number;
+  exp: number;
+  iss: null | string;
+  public_keys: string[];
+  domain_name: string;
+  manifest_uri: string;
+  redirect_uri: string;
+  version: string;
+  do_not_include_profile: boolean;
+  supports_hub_url: boolean;
+  scopes: string[]
 };
 
 
 // new ecdsa private key each time
 const authTransitKey = crypto.randomBytes(32).toString('hex');
-const authTransitPubkey = getPublicKeyFromPrivateKey(authTransitKey);
+getPublicKeyFromPrivateKey(authTransitKey);
```

Comment:
This line can just be deleted, since it looks like we don't even use this var anywhere.